docs/v2/docfx.json

```diff
@@ -0,0 +1,85 @@
+{
+  "metadata": [
+    {
+      "src": [
+        {
+          "files": [
+            "src/**.csproj"
+          ],
+          "exclude":[
+            "**/bin/**",
+            "**/obj/**",
+            "**/Examine.Test**",
+            "**/Examine.Web**"
+          ],
+		      "src": "../../"
+        }
+      ],
+      "dest": "api",
+      "disableGitFeatures": false,
+      "disableDefaultFilter": false
+    }
+  ],
+  "build": {
+    "content": [
+      {
+        "files": [
+          "api/**.yml",
+          "api/index.md"
+        ]
+      },
+      {
+        "files": [
+          "articles/**.md",
+          "articles/**/toc.yml",
+          "toc.yml",
+          "*.md",
+          "docs-v1-v2/**.md",
+          "docs-v1-v2/**/toc.yml"
+        ]
+      }
+    ],
+    "resource": [
+      {
+        "files": [
+          "images/**"
+        ]
+      }
+    ],
+    "overwrite": [
+      {
+        "files": [
+          "apidoc/**.md"
+        ],
+        "exclude": [
+          "obj/**",
+          "_site/**"
+        ]
+      }
+    ],
+    "dest": "_site",
```

Comment:
(see other comment) ... If this was _site/v2 would it deploy to a v2 directory in GH pages?

Reply:
See code as at  49a5fd3